shopyo/modules/box__default/theme/view.py

```diff
@@ -46,28 +46,31 @@
 def index():
 
     context = {}
-    themes_path = os.path.join(current_app.config["BASE_DIR"], "themes")
-    all_info = {}
-    theme_folders = get_folders(themes_path)
-    for folder in theme_folders:
-        theme_path = os.path.join(themes_path, folder)
+
+    front_themes_path = os.path.join(current_app.config["BASE_DIR"], "themes", "front")
+    all_front_info = {}
+    front_theme_folders = get_folders(front_themes_path)
+    for folder in front_theme_folders:
+        theme_path = os.path.join(front_themes_path, folder)
         info_path = os.path.join(theme_path, "info.json")
         with open(info_path) as f:
-            all_info[folder] = json.load(f)
+            all_front_info[folder] = json.load(f)
+
+    active_front_theme = get_setting("ACTIVE_FRONT_THEME")
 
-    active_theme = get_setting("ACTIVE_THEME")
-    context.update({"all_info": all_info, "active_theme": active_theme})
+    
+    context.update({"all_front_info": all_front_info, "active_front_theme": active_front_theme})
     context.update(module_settings)
 
     return render_template(
         "{}/index.html".format(module_info["module_name"]), **context
     )
 
 
-@module_blueprint.route("/activate/<theme_name>")
+@module_blueprint.route("/activate/front/<theme_name>")
 @login_required
-def activate(theme_name):
-    set_setting("ACTIVE_THEME", theme_name)
+def activate_front_theme(theme_name):
+    set_setting("ACTIVE_FRONT_THEME", theme_name)
```

Comment:
I am assuming this is to change the front end theme (previously this would change theme between boogle and mistrello). At the moment activating the frontend theme does nothing other then updating the active frontend theme in settings. Is this the intended behavior? Also how do I change the backend theme (i.e switch between boogle and mistrello)